File: API/API_MEDICAL/Entities/BenhNhan.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API_MEDICAL.Entities
{
    [Table("BENH_NHAN", Schema="PATIENT_RECORD")]
    public class BenhNhan
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        [StringLength(15)]
        public string? BHYT { get; set; }
        [Required]
        public string TEN { get; set; }
        [Required]
        public DateTime NGAY_SINH { get; set; }
        public int? TUOI { get; set; }
        [Required]
        public bool GIOI_TINH { get; set; }
        public string? NGHE_NGHIEP { get; set; }
        public string? SDT { get; set; }
        public string? DIA_CHI { get; set; }
        public string? CCCD { get; set; }
    }
}

File: App/app/src/main/java/com/example/patientrecord/BenhNhan.java
package com.example.patientrecord;

import androidx.annotation.Nullable;

import java.io.Serializable;
import java.util.Date;

public class BenhNhan implements Serializable {
    int id;
    @Nullable
    String bhyt;
    String ten;
    String ngaY_SINH;
    int tuoi;
    Boolean gioI_TINH;
    @Nullable
    String nghE_NGHIEP;
    @Nullable
    String sdt;
    @Nullable
    String diA_CHI;
    @Nullable
    String cccd;
    public BenhNhan(String ten, String ngaysinh, String bhyt, Boolean gioitinh){
        this.ten = ten;
        ngaY_SINH = ngaysinh;
        this.bhyt = bhyt;
        gioI_TINH = gioitinh;
    }
}
